docs/docs/capella/readonly.md

```diff
@@ -0,0 +1,97 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Capella read-only
+
+<!-- prettier-ignore -->
+!!! info
+    The Docker image name for this image is `capella/readonly`
+
+The read-only image builds on top of the Capella EASE remote image and provides support for the read-only use of models.
+
+It clones a git repositories and automatically injects the cloned models in the workspace.
+
+## Use the prebuilt image
+
+```
+docker run ghcr.io/dsd-dbs/capella-dockerimages/capella/readonly:$TAG
+```
+
+where `$TAG` is the Docker tag. For more information, have a look at our [tagging schema](introduction.md#tagging-schema-for-prebuilt-images).
+Please check the [`Run the container`](#run-the-container) section to get an overview over environment variables you have to set during startup.
+
+## Build it yourself
+
+### Build it manually with Docker
+
+To build the image, please run:
+
+```zsh
+docker build -t capella/readonly \
+    --build-arg BASE_IMAGE=capella/ease/remote \
+    readonly
+```
+
+## Run the container
+
+There are two options available for read-only containers.
+The first option supports multiple repositories, but you need to JSON-serizalize it.
+With the second option, you can provide the details as environment variables directly.
+
+Running the Capella read-only container is analogous to the Capella base container. Please run the [instructions of the Capella base container](../base.md#run-the-container), but add the `$EASE_LOG_LOCATION` environment variable during `docker run` (optional). The value should be the absolute path to log file. Defaults to `/proc/1/fd/1` (Docker logs) if not provided.
+
+In addition, choose one of the following two options and consider the differences.
+
+### Provide repository details as JSON
+
+With this option, any number of Git models can be loaded.
+These must first be serialized to JSON in the following format:
+
+```json
+[
+  {
+    "url": "https://github.com/DSD-DBS/py-capellambse.git", # Path that is used by 'git clone'
+    "revision": "master",
+    "depth": 1, # Optional: If depth == 0, the whole history is cloned. Defaults to 0
+    "entrypoint": "tests/data/melodymodel/5_2/Melody Model Test.aird", # Path to the aird file, starting from the root of the repository
+    "nature": "project", # Optional: Can be either 'project' or 'library'. Defaults to 'project'
+    "username": "testuser", # Optional: Only need if repository access is restricted
+    "password": "token" # Optional: Only need if repository access is restricted
+  }
+]
+```
+
+The JSON string has to be provided as value to the environment variable with the key `GIT_REPOS_JSON`:
+
+```zsh
+    -e GIT_REPOS_JSON=$GIT_REPOS_JSON
+```
+
+Please replace the `GIT_REPOS_JSON` with the JSON-string (described above).
```

Comment:
```suggestion
Please replace the `$GIT_REPOS_JSON` with the JSON-string (described above).
```

Reply:
I've removed the sentence completely, it should be clear from the context.